packages/react-compose-next/README.md

```diff
@@ -0,0 +1,30 @@
+# @fluentui/react-compose
+
+**Composition functionality for in [Fluent UI](https://aka.ms/fluent-ui/)**
+
+This functionality is in the experimental stage and **should never be used in product** as APIs might change before final release.
+
+### Usage
+
+To import `compose` and use:
```

Comment:
Maybe we should clarify that this is currently working on functional components only

Reply:
Added 👍